include/drivers/spi.h

```diff
@@ -297,7 +297,9 @@ struct spi_config {
 #define SPI_CONFIG_DT(node_id, operation_, delay_)			\
 	{								\
 		.frequency = DT_PROP(node_id, spi_max_frequency),	\
-		.operation = (operation_),				\
+		.operation = (operation_) |				\
+			DT_PROP_OR(node_id, duplex, 0) |		\
+			DT_PROP_OR(node_id, frame_format, 0),		\
```

Comment:
Since `frame-format` is defined with `default: 0` in binding, the `_OR` isn't needed.

Reply:
ok, thus the error on duplex then. I'll switch back to DT_PROP() and fix the dts bindings for duplex.